src/ui/flutter_app/lib/main.dart

```diff
@@ -37,31 +37,31 @@ import 'package:sanmill/widgets/navigation_home_screen.dart';
 import 'services/audios.dart';
 
 Future<void> main() async {
-  var catcher = Catcher(
-      rootWidget: BetterFeedback(
-        child: SanmillApp(),
-        //localeOverride: Locale(Resources.of().languageCode),
-      ),
-      ensureInitialized: true);
+  final catcher = Catcher(
+    rootWidget: BetterFeedback(
+      child: SanmillApp(),
+      //localeOverride: Locale(Resources.of().languageCode),
+    ),
+    ensureInitialized: true,
+  );
 
   String externalDirStr;
   try {
-    Directory? externalDir = await getExternalStorageDirectory();
+    final Directory? externalDir = await getExternalStorageDirectory();
     if (externalDir != null) {
-      externalDirStr = externalDir.path.toString();
+      externalDirStr = externalDir.path;
     } else {
       externalDirStr = ".";
     }
   } catch (e) {
     print(e);
     externalDirStr = ".";
   }
-  String path = externalDirStr + "/" + Constants.crashLogsFileName;
-  print("[env] ExternalStorageDirectory: " + externalDirStr);
-  String recipients = Constants.recipients;
+  final String path = "$externalDirStr/${Constants.crashLogsFileName}";
+  print("[env] ExternalStorageDirectory: $externalDirStr");
+  final String recipients = Constants.recipients;
 
-  CatcherOptions debugOptions =
-      CatcherOptions(PageReportMode(showStackTrace: true), [
```

Comment:
Can these parameters be removed because they are the default parameters? To be confirmed.